scripts/build/Dockerfile_solver_build

```diff
@@ -0,0 +1,33 @@
+ARG REPOSITORY
+ARG LLVM_SUFFIX
+ARG LLVM_VERSION_SHORT
+FROM ${REPOSITORY}/llvm_built:${LLVM_VERSION_SHORT}${LLVM_SUFFIX}
```

Comment:
Why do you need the LLVM base image for building solvers? Another candidate to be removed from the inheritance!